src/backend/tests/changeRequest.test.ts

```diff
@@ -0,0 +1,176 @@
+  reviewer: batman,
+  dateReviewed: new Date('11/24/2020'),
+  type: CR_Type.DEFINITION_CHANGE,
+  changes: [
+    {
+      crId: 2,
+      implementedChanges: [
+        {
+          changeRequestId: 2,
+          detail: 'changed cloak color from black to red',
+          implementer: {
+            email: 'notbatman@gmail.com',
+            firstName: 'Bruce',
+            lastName: 'Wayne',
+            role: 'APP_ADMIN',
+            userId: 1
+          }
+        }
+      ],
+      changeRequestId: 2,
+      implementerId: 1,
+      wbsNum: {
+        carNumber: 1,
+        dateCreated: new Date(),
+        name: 'car',
+        projectLeadId: 4,
+        projectManagerId: 5,
+        projectNumber: 2,
+        status: 'ACTIVE',
+        wbsElementId: 31,
+        workPackageNumber: 3
+      },
+      implementer: batman,
+      wbsElement: wbs,
+      wbsElementId: 65,
+      detail: 'changed cloak color from black to red'
+    }
+  ],
+  scapeChangeRequest: [
+    {
+      why: '',
+      proposedSolutions: [
+        {
+          createdBy: batman
+        }
+      ]
+    }
+  ],
+  stageGateChangeRequest: true,
+  activationChangeRequest: [
+    {
+      projectLead: batman,
+      projectManager: batman
+    }
+  ],
+  accepted: true,
+  reviewerId: 1,
+  reviewNotes: 'red is better'
+};
+
+const transformedCR = {
+  crId: 2,
+  wbsNum: { carNumber: 1, projectNumber: 2, workPackageNumber: 3 },
+  submitter: {
+    userId: 1,
+    firstName: 'Bruce',
+    lastName: 'Wayne',
+    email: 'notbatman@gmail.com',
+    emailId: 'notbatman',
+    role: 'APP_ADMIN'
+  },
+  dateSubmitted: '2020-11-25T05:00:00.000Z',
+  type: 'DEFINITION_CHANGE',
+  reviewer: {
+    userId: 1,
+    firstName: 'Bruce',
+    lastName: 'Wayne',
+    email: 'notbatman@gmail.com',
+    emailId: 'notbatman',
+    role: 'APP_ADMIN'
+  },
+  dateReviewed: '2020-11-24T05:00:00.000Z',
+  accepted: true,
+  reviewNotes: 'red is better',
+  implementedChanges: [
+    {
+      wbsNum: {
+        carNumber: 1,
+        projectNumber: 2,
+        workPackageNumber: 3
+      },
+      changeRequestId: 2,
+      implementer: {
+        email: 'notbatman@gmail.com',
+        emailId: 'notbatman',
+        firstName: 'Bruce',
+        lastName: 'Wayne',
+        role: 'APP_ADMIN',
+        userId: 1
+      },
+      detail: 'changed cloak color from black to red'
+    }
+  ]
+};
+
+describe('ChangeRequests', () => {
+  beforeEach(() => {
+    prisma.change_Request.findUnique = jest.fn();
+  });
+
+  test('getChangeRequestByID returns 404 when crId does not exist', async () => {
+    const res = await request(app).get('/123');
+
+    expect(res.statusCode).toBe(404);
+  });
+
+  test('getChangeRequestByID works', async () => {
+    jest.spyOn(prisma.change_Request, 'findUnique').mockResolvedValue(changeCloak);
+
+    const res = await request(app).get('/2');
+
+    expect(res.statusCode).toBe(200);
+    expect(res.body).toStrictEqual(transformedCR);
+  });
+});
+
+export {};
```

Comment:
Whoops, didn't see this line. Do you mind if you could delete this line and have this file until line 175? After changes, re-add, commit, and push and let me know through Slack?